### PR TITLE
Tests for crud invitation

### PR DESCRIPTION
**Pull request by KirylShalpuk**
added tests for crud invitation

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,8 +15,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.9</maven.compiler.source>
```
Comment (AlexKolonitsky) [inline pom.xml]: return

File: pom.xml [hunk 2]
```diff
@@ -30,19 +30,29 @@
         <jersey-media-moxy.version>2.28</jersey-media-moxy.version>
         <jersey-spring.version>2.28</jersey-spring.version>
         <mysql-connector.version>8.0.15</mysql-connector.version>
+        <dbunit.version>2.5.4</dbunit.version>
+        <slf4j.version>1.7.26</slf4j.version>
```
Comment (AlexKolonitsky) [inline pom.xml]: revert

File: src/test/java/app/dao/impl/Invitation/DeleteInvitationDaoTest.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import java.sql.Date;
+import java.sql.SQLException;
+
+public class DeleteInvitationDaoTest extends InitilizationInvitationDaoTest {
+
+    @Test
+    public void deleteById() {
+        basicCrudDao.deleteById(4);
+        try {
+            IDataSet iDataSet = new FlatXmlDataSetBuilder().build(getClass().getClassLoader()
```
Comment (AlexKolonitsky) [inline src/test/java/app/dao/impl/Invitation/DeleteInvitationDaoTest.java]: common method

File: src/test/java/app/dao/impl/Invitation/GetFinfByIdInvitationDaoTest.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                    String.valueOf(invitation.getDateEnd()));
+            Assert.assertEquals(iTable.getValue(NUMBER_OF_FIRST_ROW, "status").toString(),
+                    String.valueOf(invitation.getStatus()));
+        } catch (DataSetException e) {
+            e.printStackTrace();
+        }
+    }
+
+//    @Test(expected = EntityNotFoundException.class)
+//    public void getFindByIdException() {
```
Comment (AlexKolonitsky) [inline src/test/java/app/dao/impl/Invitation/GetFinfByIdInvitationDaoTest.java]: remove

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (KirylShalpuk) [reply pom.xml]: completed

File: src/test/java/app/dao/impl/Invitation/GetFinfByIdInvitationDaoTest.java (hunk 4 above)
Comment (KirylShalpuk) [reply src/test/java/app/dao/impl/Invitation/GetFinfByIdInvitationDaoTest.java]: completed

File: pom.xml (hunk 2 above)
Comment (KirylShalpuk) [reply pom.xml]: completed

File: src/test/java/app/dao/impl/Invitation/DeleteInvitationDaoTest.java (hunk 3 above)
Comment (KirylShalpuk) [reply src/test/java/app/dao/impl/Invitation/DeleteInvitationDaoTest.java]: completed

File: pom.xml [hunk 5]
```diff
@@ -291,7 +291,7 @@
                     <changeLogFile>liquibase/db-changelog-master.xml</changeLogFile>
                 </configuration>
             </plugin>
-      
+
```
Comment (AlexKolonitsky) [inline pom.xml]: remove changes

File: pom.xml [hunk 6]
```diff
@@ -315,4 +315,4 @@
         </plugins>
     </reporting>
 
-</project>
\ No newline at end of file
+</project>
```
Comment (AlexKolonitsky) [inline pom.xml]: remove changes

File: src/main/java/app/config/JerseyConfig.java [hunk 7]
```diff
@@ -18,6 +18,5 @@ public JerseyConfig() {
         register(NotificationResource.class);
         register(InvitationResource.class);
         register(TimesheetResource.class);
-
     }
```
Comment (AlexKolonitsky) [inline src/main/java/app/config/JerseyConfig.java]: remove changes

File: src/test/resources/project.properties [hunk 8]
```diff
@@ -2,8 +2,11 @@ version=${project.version}
 artifactId=${project.artifactId}
```
Comment (AlexKolonitsky) [inline src/test/resources/project.properties]: remove changes

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: src/main/java/app/config/JerseyConfig.java (hunk 7 above)
Comment (KirylShalpuk) [reply src/main/java/app/config/JerseyConfig.java]: completed

File: pom.xml (hunk 6 above)
Comment (KirylShalpuk) [reply pom.xml]: completed

File: pom.xml (hunk 5 above)
Comment (KirylShalpuk) [reply pom.xml]: completed

**Comment by KirylShalpuk** [comment]
completed correct yours remarks

**Pull request closed without merging**